src/patternfly/components/Form/form.scss

```diff
@@ -261,7 +255,6 @@ $pf-v5-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "l
 }
 
 .#{$form}__group-label-info {
-  margin-inline-start: var(--#{$form}__group-label-info--MarginLeft);
```

Comment:
Did you mean to remove this? You still have the var, and without this declaration the text and info text touch

<img width="840" alt="Screenshot 2023-12-21 at 1 39 37 PM" src="https://github.com/patternfly/patternfly/assets/35148959/def161b3-ed71-41ad-8076-179947f08287">